src/utils/isTokenValid.js

```diff
@@ -0,0 +1,32 @@
+import base64 from 'base-64';
+
+const isTokenValid = (token) => {
+  if (!token) return false;
+  const keys = ['header', 'payload', 'signature'];
+
+  const parts = token.split('.');
+  if (parts.length !== 3) return false;
```

Comment:
A JWT can also not have a signature in some cases, but never in our case. (So this is okay 👍) More of an FYI why the other library might not have complained about that.

Reply:
The other library only looked at the payload, but not the header. The header should also be valid JSON.